recipes_source/distributed_checkpoint_recipe.rst

```diff
@@ -161,6 +187,36 @@ The reason that we need the ``state_dict`` prior to loading is:
     CHECKPOINT_DIR = "checkpoint"
 
 
+    class AppState(Stateful):
```

Comment:
Isn't this the same as lines 70-99?

Reply:
Yep @svekars , I thought it would be better if each code block was self contained. If that's not the case, I can remove duplicates

Reply:
I'll leave it up to you to decide. I just wanted to make sure it wasn't copy pasted twice by mistake.